pages/apim/3.x/event-native/apim-event-native-api-management-activate-mode.adoc

```diff
@@ -0,0 +1,30 @@
+[[apim-event-native-api-management-activate-mode]]
+= Activating/deactivating the new execution engine mode
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_event_native_api_management_activate_mode.html
+:page-folder: apim/event-native
+:page-layout: apim3x
+
+[label label-version]#New in version 3.19.0#
+
+To activate the new event-native policy execution mode, you must enable it in the `gravitee.yml` file using the following new keys:
+
+* `JUPITER_MODE_ENABLED`. This option enables you to activate or deactivate the new event-driven execution engine mode on the Gateway. The key values can be `true` or `false`, respectively.
```

Comment:
I remember someone say Jupiter is an internal work for this engine. Shouldn't we discuss about the officiel one?

Reply:
@mouligno @jhaeyaert any thoughts on this - what should we change this with if there is a new value for external use?

Reply:
We didn't change this value so we should keep it.

Reply:
Thanks - one to watch for then (and update the doc).